tests/nemo_text_processing/de/data_text_normalization/test_cases_word.txt

```diff
@@ -1,6 +1,5 @@
 ~
 yahoo!~yahoo!
-zwanzig!~20!
```

Comment:
could you add this one back?

Reply:
deterministic requires a space between zwanzig and !

Reply:
why? current Eng TN work without it https://github.com/NVIDIA/NeMo/blob/main/tests/nemo_text_processing/en/data_text_normalization/test_cases_word.txt#L8

Reply:
ok added back